src/components/FlatmapVuer.vue

```diff
@@ -15,61 +16,96 @@
+          <el-popover content="Find these markers for data" placement="right"
+            :appendToBody=false trigger="manual" popper-class="flatmap-popper popper-bump-right" v-model="hoverVisibilities[5].value" ref="markerPopover">
+          </el-popover>
+          <div v-show="hoverVisibilities[5].value" class="flatmap-marker-help" v-html="flatmapMarker" v-popover:markerPopover></div>
+          <el-row>
+            <el-col :span="12">
+              <div class="pathways-display-text">
+                Pathways
+              </div>
+            </el-col>
+            <el-col :span="12">
+              <el-checkbox class="all-checkbox" :indeterminate="isIndeterminate" v-model="checkAll" @change="handleCheckAllChange">Display all</el-checkbox>
+            </el-col>
+          </el-row>
+          <el-checkbox-group v-model="checkedItems" size="small" 
+            class="checkbox-group" @change="handleCheckedItemsChange">
+            <div class="checkbox-group-inner">
+              <el-row v-for="item in pathways" :key="item.type" :label="item.type">
+                <div class="checkbox-container">
+                  <el-checkbox
+                    class="my-checkbox"
+                    :label="item.type"
+                    @change="visibilityToggle()"
+                    :checked="true"
+                  ><div class="path-visual" :class="item.type"></div>{{item.label}}</el-checkbox>
+                </div>
+              </el-row>
+            </div>
+          </el-checkbox-group>
+        </div>
+        <div @click="toggleDrawer" class="drawer-button" :class="{ open: drawerOpen, close: !drawerOpen }">
+          <i class="el-icon-arrow-left"></i>
+        </div>
+      </div>
+      <el-popover
+        ref="backgroundPopover"
+        placement="top-start"
+        width="128"
+        :appendToBody=false
+        trigger="click"
+        popper-class="background-popper">
+        <el-row class="backgroundText">
+          Change background
+        </el-row>
+        <el-row class="backgroundChooser" >
+          <div v-for="item in availableBackground" :key="item" 
+            :class="['backgroundChoice', item, item == currentBackground ? 'active' :'']" 
+            @click="backgroundChangeCallback(item)"/>
+        </el-row>
+      </el-popover>
+      <el-popover  content="Change background color" placement="right" v-model="hoverVisibilities[3].value"
+        :appendToBody=false trigger="manual" popper-class="flatmap-popper">
+        <SvgIcon v-popover:backgroundPopover icon="changeBckgd" class="icon-button background-colour" 
+          :class="{ open: drawerOpen, close: !drawerOpen }" slot="reference"
+          @mouseover.native="showToolitip(3)" @mouseout.native="hideToolitip(3)"/>
+      </el-popover>
     </div>
   </div>
 </template>
 
 <script>
 /* eslint-disable no-alert, no-console */
 import Vue from "vue";
+import { SvgIcon, SvgSprite} from '@abi-software/svg-sprite'
```

Comment:
I'm sure you know this but I'm using the colour sprite at this stage like so: 

`main.js`
```
import {SvgSprite, SvgIcon, SvgSpriteColor} from '@abi-software/svg-sprite'

Vue.component('svg-icon', SvgIcon)
Vue.component('svg-sprite', SvgSprite)
Vue.component('svg-sprite-color', SvgSpriteColor)
```

Reply:
Flatmapvuer does not use any of the problematic icons but I can switch it over.